docs/developer-interface.rst

```diff
@@ -172,8 +172,16 @@ Project
 .. autoclass:: tamr_unify_client.project.resource.Project
   :members:
 
+<<<<<<< HEAD
```

Comment:
The attribute mapping section should probably be put under project, and git merge stuff needs to be removed